### PR TITLE
Fixes to build process

### LINKED ISSUE #22: No /usr/lib/initcpio/hooks/archiso on docker build
When i'm running docker build it gives me this error.
`cp: cannot stat '/usr/lib/initcpio/hooks/archiso': No  such file or directory`

### PR DESCRIPTION
**Pull request by moonwolf-github**
Added lynx to docker image, changed paths and packages parsing in build.sh
closes #22

### CODE REVIEW

File: gameros/build.sh [hunk 1]
```diff
@@ -75,12 +75,12 @@ make_setup_mkinitcpio() {
     mkdir -p ${work_dir}/x86_64/airootfs/etc/initcpio/hooks
     mkdir -p ${work_dir}/x86_64/airootfs/etc/initcpio/install
     for _hook in archiso archiso_shutdown archiso_pxe_common archiso_pxe_nbd archiso_pxe_http archiso_pxe_nfs archiso_loop_mnt; do
-        cp /usr/lib/initcpio/hooks/${_hook} ${work_dir}/x86_64/airootfs/etc/initcpio/hooks
-        cp /usr/lib/initcpio/install/${_hook} ${work_dir}/x86_64/airootfs/etc/initcpio/install
+        cp ${work_dir}/x86_64/airootfs//usr/lib/initcpio/hooks/${_hook} ${work_dir}/x86_64/airootfs/etc/initcpio/hooks
```
Comment (sharkwouter) [inline gameros/build.sh:78]: Looks a bit better without the double / after airootfs

**Comment by sharkwouter** [comment]
Thanks for this. I can't really formally review this at the moment, but I did add on comment which is actually just a nitpick.

File: gameros/build.sh (hunk 1 above)
Comment (moonwolf-github) [reply gameros/build.sh]: Should be OK now.

**Comment by sharkwouter** [comment]
I've tested this and it results in an ISO which works in Docker.  To test this I ran the build-iso-docker.sh script on Debian and tested the resulting ISO in Virtualbox. It boots and installs fine. Maybe if the initcpio hooks are included correctly will need to be tested on real hardware.

**Comment by moonwolf-github** [comment]
I successfully ran it on real hardware (i'm making 5.9 kernel version which works already). Of course it's not comprehensive testing by any means.

**Pull request merged**